src/options.h

```diff
@@ -1884,11 +1884,18 @@ donot_indent_func_def_close_paren;
 ///////////////////////////////////////////////////////////////////////////////
 //BEGIN Newline adding and removing options
 
-// Whether to collapse empty blocks between '{' and '}'.
-// If true, overrides nl_inside_empty_func
+// Whether to collapse empty blocks between '{' and '}' except for functions.
+// Use nl_collapse_empty_body_functions to specify how empty function braces
+// should be formatted.
 extern Option<bool>
 nl_collapse_empty_body;
 
+// Whether to skip collapsing empty blocks between '{' and '}' for
```

Comment:
The description and the functionaliy of the option are opposite. 
"skip collapsing" implies that if the option is 'true', then empty function bodies will not be collapsed. But if the option is 'true' it does in fact collapse the body.
The description should be reworded in a way similar to the one of `nl_collapse_empty_body`, that is "Whether to collapse empty function blocks ...."